patches_for_WebRTC_org/m84/src/modules/audio_device/win/audio_device_core_win.h

```diff
@@ -32,9 +34,10 @@
       goto Cleanup; \
   } while (false)
 
-HRESULT WaitForASyncWithEvent(_In_ IAsyncInfo* async_info,
-                              _In_ HANDLE event_completed_handle,
-                              _In_ DWORD timeout_ms);
+HRESULT WaitForASyncWithEvent(
+    _In_ IAsyncInfo* async_info,
```

Comment:
Please use the same indentation pattern as the rest of the file.

Reply:
It should be clang automatic indentation: as `_In_ std::shared_ptr<HANDLE> event_completed_handle_ptr,` is pretty long it already wraps first argument on a new line.